src/AVFoundation/AVLayerVideoGravity.cs

```diff
@@ -57,7 +59,7 @@ static internal AVLayerVideoGravity KeyToEnum (NSString key, bool clamp = true)
 			return (AVLayerVideoGravity) (-1);
 		}
 
-		static internal NSString EnumToKey (AVLayerVideoGravity vg)
+		static internal NSString EnumToKey (AVLayerVideoGravity? vg)
```

Comment:
Why can't you return NSString? and remove the null!

Reply:
@mandel-macaque The problem with adding the nullable return is that properties coming from src/avfoundation.cs that are not nullable are being assigned with EnumToKey. We looked at adding the default value for GravityResizeAspect, but @rolfbjarne pointed out [here](https://github.com/xamarin/xamarin-macios/pull/14134#pullrequestreview-901359406) that VideoGravity-type objects are not the only assignments made by EnumToKey and that it is just better to leave as is.